WMF Framework/FeatureFlags.swift

```diff
@@ -10,4 +10,11 @@ public struct FeatureFlags {
         return true
     }
     
+    public static var applePayEnabled: Bool {
```

Comment:
Minor, but what do you think about something like `applePayDonationEnabled` instead? On first read `applePayEnabled` connoted the availability of Apple Pay _itself_ to me, rather than our intent of Apple Pay as a donation option.

Reply:
Done (twice actually, sorry). I went with the more general `donorExperienceImrovementsEnabled` - I'm hoping to be able to wrap up all of our improvements in the next few weeks around that for easy rollback, not just the native form.

Reply:
👍🏽 new name works great, much more precise about what it is